src/Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv/Internal/Infrastructure/EventSourceExtensions.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Server.Kestrel.Internal.Http;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Internal.Infrastructure
+{
+    public static class EventSourceExtensions
+    {
+        public static void ConnectionStart(this KestrelEventSource eventSource, Connection connection)
```

Comment:
Since you are moving KestrelEventSource back to the Core project, can this change also move the connection start/stop logging to core? It would be nice if this tracing could be shared between transports.